src/collective/fingerpointing/upgrades/profiles/1_to_2/registry.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
You mean without an upgrade step? That would mean people have to reinstall this addon, which means they loose theier settings?